matcalc/elasticity.py

```diff
@@ -21,29 +24,32 @@ class ElasticityCalc(PropCalc):
     def __init__(
         self,
         calculator: Calculator,
-        norm_strains: float = 0.01,
-        shear_strains: float = 0.01,
+        norm_strains=(0.001, 0.003, 0.005, 0.01),
+        shear_strains=(0.001, 0.003, 0.005, 0.01),
```

Comment:
Can you comment how this change affects run time of the calculator? Maybe post a quantitative comparison of `main` branch vs your PR run time.

Reply:
This really depends on structure. There are 3 stress calls per strain in `norm_strains` and `shear_strains`, plus one for the equilibrium structure, and this does not change over structures. Usually, if a relaxation is done and the structure is more complex, the relaxation will take up the vast majority of the time anyway.

For Li2O with M3GNet, with one norm strain, one shear strain, and no relaxation (equivalent to `main`), it takes 0.35 seconds. For four of each type of strain, it takes 1.33 seconds. (The relaxation takes about 0.1 seconds). But for LiFePO4, one of each type on my computer takes 0.95 seconds and four of each type takes 3.36 seconds, while the relaxation seems to be much more expensive.